internal/runners/reset/reset.go

```diff
@@ -56,36 +55,28 @@ func (r *Reset) Run() error {
 		return locale.NewInputError("err_reset_latest", "You are already on the latest commit")
 	}
 
-	r.out.Print(locale.Tl("reset_commit", "Your project will be reset at [ACTIONABLE]{{.V0}}[/RESET]\n", latestCommit.String()))
+	r.out.Print(locale.Tl("reset_commit", "Your project will be reset to [ACTIONABLE]{{.V0}}[/RESET]\n", latestCommit.String()))
 
-	originalCommitID := r.project.CommitID()
-	revertCommit, err := model.GetRevertCommit(strfmt.UUID(originalCommitID), *latestCommit)
-	if err != nil {
-		return locale.WrapError(err, "err_revert_refresh")
-	}
-
-	changesMade := len(revertCommit.Changeset) > 0
-	if changesMade {
-		changes := commit.FormatChanges(revertCommit)
-		r.out.Print(locale.Tl("reset_changes", "Changes to local instance:"))
-		r.out.Print(changes)
-		r.out.Print("")
-	}
-
-	confirm, err := r.prompt.Confirm("", locale.Tl("reset_confim", "Continue?"), new(bool))
+	confirm, err := r.prompt.Confirm("", locale.Tl("reset_confim", "Resetting is destructive, you will loose any changes that were not pushed. Are you sure you want to do this?"), new(bool))
 	if err != nil {
 		return locale.WrapError(err, "err_reset_confirm", "Could not confirm reset choice")
 	}
 	if !confirm {
 		return locale.NewInputError("err_reset_aborted", "Reset aborted by user")
 	}
 
+	originalCommitID := r.project.CommitID()
 	err = r.project.Source().SetCommit(latestCommit.String(), r.project.IsHeadless())
 	if err != nil {
 		return locale.WrapError(err, "err_reset_set_commit", "Could not update commit ID")
 	}
 
-	err = runbits.RefreshRuntime(r.auth, r.out, r.project, r.config.CachePath(), *latestCommit, changesMade)
+	revertCommit, err := model.GetRevertCommit(strfmt.UUID(originalCommitID), *latestCommit)
+	if err != nil {
+		return locale.WrapError(err, "err_revert_refresh")
+	}
+
+	err = runbits.RefreshRuntime(r.auth, r.out, r.project, r.config.CachePath(), *latestCommit, len(revertCommit.Changeset) > 0)
```

Comment:
```suggestion
	revertCommit, err := model.GetRevertCommit(strfmt.UUID(originalCommitID), *latestCommit)
	if err != nil {
		return locale.WrapError(err, "err_revert_refresh")
	}

	err = runbits.RefreshRuntime(r.auth, r.out, r.project, r.config.CachePath(), *latestCommit, true)
```

We already know by the action that someone is taking that we'll want to pull in changes.